.github/workflows/tests-on-push.yml

```diff
@@ -25,19 +23,15 @@ jobs:
       - name: Install java
         uses: actions/setup-java@v3
         with:
-          distribution: 'temurin'
-          java-version: '17'
+          distribution: "temurin"
```

Comment:
Are double quotes handled differently?

Reply:
It handled in same way. Formatter change it to double quotes. Anyway i've returned single quotes